### ISSUE TITLE
Different animations for different special workspaces

### ISSUE THREAD

**Issue by Fustigate8933** [hyprwm/Hyprland#4807]
### Description

Hi, let's say I have 2 special workspaces. How do I set it in hyprland.conf so that one special workspace slides in from the bottom and the other slides in from the right? Currently, I have this setting, but it applies to every special workspace: animation = specialWorkspace, 1, 5.5, myBezier, slidevert.

**Comment by icejix** [comment]
I agree, I'm trying to bind a special button on my mouse to pull up a new special workspace, which instead flies from the top instead of the bottom. Would like to see this added at some point, hopefully soon.

**Comment by vaxerski** [comment]
Hello there!

This issue has been closed, as we are moving from issues to discussions. More about this can be found [here](https://wiki.hyprland.org/Contributing-and-Debugging/Issue-Guidelines/) and in #9854.

Firstly, please make sure this issue is still relevant in latest Hyprland. If so, we ask you to open a [discussion](https://github.com/hyprwm/Hyprland/discussions) (please read the discussion guidelines first, in the pinned post)

Thank you for your understanding!
